source/js/components/validation.js

```diff
@@ -0,0 +1,43 @@
+// constants for "Other Amount" inputs and error messages
+const oneTimeOtherAmountInput = document.querySelector(
+  ".one-time-amount-other-input"
+);
+const monthlyOtherAmountInput = document.querySelector(
+  ".monthly-amount-other-input"
+);
+const oneTimeOtherAmountErrorMessage = document.querySelector(
+  ".error-message__one-time-other-amount"
+);
+const monthlyOtherAmountErrorMessage = document.querySelector(
+  ".error-message__monthly-other-amount"
+);
+
+// Handler for "other amount" inputs
+function otherAmountInputValidation() {
+  oneTimeOtherAmountInput.addEventListener("blur", (e) => {
+    if (document.querySelector(".one-time-donation-amount-radio:checked")) {
+      const attemptedValue = oneTimeOtherAmountInput.value;
+      inputValueCheck(attemptedValue, oneTimeOtherAmountErrorMessage);
+    }
+  });
+  monthlyOtherAmountInput.addEventListener("blur", (e) => {
+    if (document.querySelector(".monthly-donation-amount-radio:checked")) {
+      const attemptedValue = monthlyOtherAmountInput.value;
+      inputValueCheck(attemptedValue, monthlyOtherAmountErrorMessage);
+    }
+  });
+}
+
+function inputValueCheck(valueToCheck, messageToUpdate) {
+  const number = parseFloat(valueToCheck);
+  // Checking if attempted value is a number and >= 2
+  if (!isNaN(number) && number && number >= 2) {
```

Comment:
note that we have _lots_ of different currencies, so while for US$ the minimum is 2, if someone picks a different currency, this logic needs to tap into that currency's minimum value instead.

Reply:
Hi Pomax, thanks for the catch! I have updated this file to now validate that the attempted donation amount is between the input's minimum and maximum value. Anything else, and the error message "invalid amount entered" will appear, and a tooltip will also let the user know how to fix it (ex: "Value must be higher than x").